modules/release-notes/con-breaking-changes.adoc

```diff
@@ -1,33 +1,4 @@
 [id='con-breaking-changes_{context}']
 = Breaking changes
 
-This section lists breaking changes with {product} {product-version}:
-
-== Configuration change in the GitHub Organization catalog provider  
-
-The GitHub Organization catalog module for the new backend system switched to a new `GithubMultiOrgEntityProvider` provider, which allows ingesting data from multiple GitHub organizations. This new provider brings in changes to the default plugin configuration settings. You must upgrade the `@backstage/plugin-catalog-backend-module-github-org` plugin and reconfigure it in the following manner:
-
-.Configuring the `@backstage/plugin-catalog-backend-module-github-org` plugin
---
-[source,yaml]
-----
-kind: ConfigMap
-apiVersion: v1
-metadata:
-  name: app-config-rhdh
-data:
-  app-config-rhdh.yaml: |
-    ...
-    catalog:
-      providers:
-        githubOrg:
-          id: production
-          githubUrl: "${GITHUB_URL}"
-          orgs: [ "${GITHUB_ORG}" ] <1>
-    ...
-----
-
-<1> Replace `${GITHUB_ORG}` with the GitHub organization you want to ingest users from.
---
-
-For more information, see the link:{LinkGettingStartedGuide}#enabling-github-organization-member-discovery-in-red-hat-developer-hub[Enabling GitHub organization member discovery in {product}] section in the _Getting started guide_.
+This section lists breaking changes with {product} {product-version}:
```

Comment:
some breaking changes to list here:
* https://issues.redhat.com/browse/RHIDP-2044 - which has an RN text.  should call this out as a breaking change because users need auth on backend endpoints now. 
* https://issues.redhat.com/browse/RHIDP-2043 - guest sign in.  We should add an RN text explaining the changes in case a customer wants to run in development mode for test/debug purposes. 
* https://github.com/backstage/backstage/blob/237d7e832541fb990bdee0bd5af1840c3b790e1b/docs/releases/v1.27.0-changelog.md
* https://github.com/backstage/backstage/releases/tag/v1.27.0